das-cli/src/common/bus_node/busnode_command_registry.py

```
@@ -50,58 +48,63 @@ def _gen_default_cmd(self, service, endpoint, ports_range):
 
         return f"busnode --service={service} --endpoint={endpoint} --ports-range={ports_range} {db_flag} --config={CURRENT_CONFIGFILE_PATH}".strip()
 
-    def _cmd_atomdb_broker(self, service, endpoint, ports_range, options, **args):
-        base = self._gen_default_cmd(service, endpoint, ports_range)
+    def _get_bus_endpoint(self, options):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_get_bus_endpoint`

Add return type annotation: `str`

(ANN202)